### ISSUE TITLE
How to perform effective PT in LLaMA-2-13B and CNN/DM

### ISSUE THREAD

**Issue by junzhang-zj** [google-research/prompt-tuning#297]
Great job and warehouse! I set 20 virtual tokens, used lr=1e-2, and randomly selected 3k instances from the CNN/DM training set to prompt tuning LLaMA-2-13B. I've seen a steady decrease in PPL (around 2.7) on both the training and validation sets. However, its rouge score did not improve significantly on the test set. Do you have any suggestions for changing the settings of effective PT?
